lib/ansible/modules/net_tools/cloudflare_dns.py

```diff
@@ -473,7 +473,8 @@ def delete_dns_records(self, **kwargs):
         content = params['value']
         search_record = params['record']
         if params['type'] == 'SRV':
-            content = str(params['weight']) + '\t' + str(params['port']) + '\t' + params['value']
+            if not (params['value'] is None or params['value'] == ''):
```

Comment:
Can we simplify this as - 

```
if params.get('value') not in ['', None]:
```

Reply:
Yeah, that does seem like a simpler/better syntax.

On the other hand, `if (foo is None) or (foo == '')` is the syntax which is used everywhere else in that module, and consistency is a good thing.

So, what is more important, used the better syntax or the more consistent syntax?

Reply:
I would prefer consistent syntax.